pyfolio/tests/test_perf_attrib.py

```diff
@@ -28,7 +28,7 @@ def generate_toy_risk_model_output(start_date='2017-01-01', periods=10,
     dts = pd.date_range(start_date, periods=periods)
     np.random.seed(123)
     tickers = ['AAPL', 'TLT', 'XOM']
-    styles = ['factor{}'.format(i) for i in xrange(num_styles)]
+    styles = ['factor{}'.format(i) for i in range(num_styles)]
```

Comment:
Did this just not ever pass the python 3 tests?

Reply:
yeah, it would fail on the flake8 call